packages/server/README.md

```diff
@@ -106,10 +106,25 @@ If the provider you're looking for hasn't been created yet, see the [develop a p
 
 Once you've added a provider as a dependency, it can be registered with OpenFeature like this:
 
+#### Awaitable
+
+To register a provider and ensure it is ready before further actions are taken, you can use the `setProviderAndWait` method as shown below:
+
+```ts
+await OpenFeature.setProviderAndWait(new MyProvider());
+```  
+
+#### Synchronous
+
+To register a provider in a synchronous manner, you can use the `setProvider` method as shown below:
+
 ```ts
-OpenFeature.setProvider(new MyProvider())
+OpenFeature.setProvider(new MyProvider());
 ```
 
+> [!NOTE]  
+> The status of the provider can be tracked using [events](#eventing).
+
```

Comment:
Just outta curiosity, how do these sort of notes render in docusaurus?

Reply:
They don't work... it appears GitHub make up this syntax and it's not widely supported. Removing the fancy rendering for now.

Reply:
To do the same thing in docusaurus you can use 
```markdown
:::note

The status of the provider can be tracked using [events](#eventing).
```

https://docusaurus.io/docs/next/markdown-features/admonitions